ros/config/common_costmap_params.yaml

```diff
@@ -10,7 +10,7 @@ observation_sources: laser_scan_sensor
 laser_scan_sensor:
     sensor_frame: /rslidar_projected_front
     data_type: LaserScan
-    topic: /projected_scan
+    topic: /projected_scan_front
```

Comment:
Same as above; we should use the full scan if the robot doesn't have a cart attached and switch to only the front if it does have a cart.